app/services/model_exporters/experiment_exporter.rb

```diff
@@ -35,10 +37,17 @@ def export_template_to_dir
     end
 
     def experiment
+      if @include_archived
+        my_modules = @experiment.my_modules
+        my_module_groups = @experiment.my_module_groups
+      else
+        my_modules = @experiment.active_my_modules
```

Comment:
You should scope `my_modules` like this... `@experiment.my_modules.active` instead of using a method `active_my_modules`